draft-packages/tabs/KaizenDraft/Tabs/styles.scss

```diff
@@ -69,18 +60,34 @@
       }
     }
   }
-
-  &:hover {
-    @extend %tabHover;
-  }
 }
 
-.tab + .tab {
-  @include ca-margin($start: $ca-grid * 0.5);
+.row {
+  composes: tab;
+  font-weight: $kz-typography-heading-4-font-weight;
+  border-top-left-radius: $kz-border-borderless-border-radius;
+  border-top-right-radius: $kz-border-borderless-border-radius;
+  padding: 0 $ca-grid * 0.75;
+  height: 3 * $ca-grid;
+
+  top: 0; // this overrides baseline shift. @include ca-inherit-baseline does not work because
+  // we need to make this position: relative, for the absolutely positioned underline.
+
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  min-width: 4.5rem - ($ca-grid * 0.75) * 2; // give tabs with short text some breathing room
+
+  :global(.js-focus-visible) &:global(.focus-visible):after {
+    border-radius: $kz-border-borderless-border-radius
+      $kz-border-borderless-border-radius 0 0;
+  }
 }
 
-.activeTab {
+.activeTabRow {
```

Comment:
If `.row` is renamed to `.rowTab` then this would be better as `activeRowTab`.

Otherwise, here's a completely different suggestion.  I think you've got a bit of a BEM thing going one where the last word is the "modifier", but that's hard to grok that without any separators - without it you tend to read it as plain english, which is where it gets confusing. I believe `_`'s are valid - maybe that could be used as a separator?
So, what about `.tab_row` and `.tab_column`, `.activeTab_row`, `.activeTab_column`. I find that aesthetically ugly (that really doesn't matter), but a lot easier to grok.

Reply:
Yup, managing these modifiers are going to start grinding away at me until we're a bit more consistent. 

I don't think i've made this any _worse_, but I definitely haven't made it _better_. I didn't change it too much from what it was - it was previously `stateThing` (like`activeTab`, `disabledTab`) now it's `thingState`, like `rowTabDisabled` )